test/lit/typed_continuations_resume.wast

```diff
@@ -0,0 +1,94 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+
+
+;; RUN: wasm-as %s -all -g -o %t.wasm
+;; RUN: wasm-dis %t.wasm -all -o %t.wast
+;; RUN: wasm-as %s -all -o %t.nodebug.wasm
+;; RUN: wasm-dis %t.nodebug.wasm -all -o %t.nodebug.wast
+;; RUN: wasm-opt %t.wast -all -o %t.text.wast -g -S
+;; RUN: cat %t.wast | filecheck %s --check-prefix=CHECK-BINARY
+;; RUN: cat %t.nodebug.wast | filecheck %s --check-prefix=CHECK-NODEBUG
+;; RUN: cat %t.text.wast | filecheck %s --check-prefix=CHECK-TEXT
+(module
+ ;; CHECK-BINARY:      (type $ft (func (param i32) (result i32)))
+ ;; CHECK-TEXT:      (type $ft (func (param i32) (result i32)))
+ (type $ft (func (param i32) (result i32)))
+ ;; CHECK-BINARY:      (type $ct (cont $ft))
+ ;; CHECK-TEXT:      (type $ct (cont $ft))
+ (type $ct (cont $ft))
+ ;; CHECK-BINARY:      (type $2 (func (result i32)))
+
+ ;; CHECK-BINARY:      (type $3 (func (param (ref $ct)) (result i32)))
+
+ ;; CHECK-BINARY:      (tag $t (result i32))
+ ;; CHECK-TEXT:      (type $2 (func (result i32)))
+
+ ;; CHECK-TEXT:      (type $3 (func (param (ref $ct)) (result i32)))
+
+ ;; CHECK-TEXT:      (tag $t (result i32))
+ (tag $t (result i32))
```

Comment:
Can we have a test with `resume` using a tag with non-empty `params` as well?

Reply:
Done, I've just changed `$t` in this test.